spec/player.spec.js

```diff
@@ -0,0 +1,103 @@
+var Player = require("../player.js");
+
+describe('Player', function() {
+  var azula = new Player("Princess Azula");
+
+  it('has a name', function() {
+    expect(azula.name).toEqual(jasmine.any(String));
+  });
+
+  it('has the correct name', function() {
+    expect(azula.name).toEqual("Princess Azula");
+  });
+
+  it('has an empty array of played words to start', function() {
+    expect(azula.plays).toEqual([]);
+  });
+});
+
+describe('play(word)', function() {
+  var toph = new Player("Toph")
+  toph.play("kittens");
+
+  it('adds a word to plays property', function() {
+    expect(toph.plays.length).toEqual(1);
+  });
+
+  it('adds the correct word to plays property', function() {
+    expect(toph.plays[0]).toEqual("kittens");
+  });
+
+  it('returns false if player has already won', function() {
+    toph.play("puppies")
+    expect(toph.play("zebra")).toEqual(false);
+  });
```

Comment:
I'm terrible at testing but is this test actually testing what you think it is? I'm a little confused about what is happening here.